x-pack/filebeat/input/cel/input.go

```diff
@@ -176,6 +177,16 @@ func (i input) run(env v2.Context, src *source, cursor map[string]interface{}, p
 			Value: cfg.Auth.Token.Value,
 		}
 	}
+
+	if cfg.Auth.AWS.IsEnabled() {
+		tr, err := aws.InitializeSingerTransport(*cfg.Auth.AWS, log, client.Transport)
+		if err != nil {
+			log.Errorw("aws auth method: initialize of aws config failed for aws signer", "error", err)
```

Comment:
```suggestion
			log.Errorw("initialize of aws config failed for aws signer", "error", err)
```
There's enough text to find this without the colon prefix.